test/reducer.passage.test.js

```diff
@@ -251,4 +252,57 @@ describe('passageReducer', () => {
       });
     });
   });
+
+  describe('save and restore state', () => {
+    let initialState = {};
+
+    beforeEach(() => {
+      initialState = {
+        active: {
+          VERSE_MODE: 0,
+          SEGMENT_MODE: 0,
+          CHAPTER_MODE: 0
+        },
+        mode: VERSE_MODE,
+        recallStage: RECALL_STAGES.FULL,
+        isAudioPlaying: false
+      };
+      clearSavedTestState();
+    });
+
+    it('should restore last state.active', () => {
```

Comment:
I think we should include one more test case to make sure the state is stored to the cookie on subsequent state changes.